src/sentry/integrations/slack/views/unlink_team.py

```diff
@@ -51,32 +56,59 @@ class SlackUnlinkTeamView(BaseView):
     Django view for unlinking team from slack channel. Deletes from ExternalActor table.
     """
 
+    _METRICS_SUCCESS_KEY = "sentry.integrations.slack.unlink_team_view.success"
+    _METRICS_FAILURE_KEY = "sentry.integrations.slack.unlink_team_view.failure"
```

Comment:
fyi i'm collecting all slack metrics in `src/sentry/integrations/slack/metrics.py` if you want to put these metrics in there, it's easier to see all of them at once that way

Reply:
thanks for the heads up, ill update some of the other metrics i made as well